tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -169,15 +169,16 @@ public function theJsonDataFromLastResponseShouldMatch(
 
 	/**
 	 * @param string $subject
+	 * @param string $resource
 	 *
 	 * @return object
 	 */
-	public function filterResponseAccordingToNotificationSubject(string $subject): object {
+	public function filterResponseAccordingToNotificationSubject(string $subject, string $resource): object {
```

Comment:
This function is used by another step too where we might not need the resource check